packages/terra-table/src/proptypes/cellShape.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+const cellShape = PropTypes.shape({
+  /**
+   * Content that will be rendered within the Cell.
+   */
+  content: PropTypes.node,
+  /**
+   *  Boolean indicating if cell contents are masked.
+   */
+  isMasked: PropTypes.bool,
```

Comment:
The cell shape still does not have all of its props from the DataGrid package implementation.